src/global/ValueId.h

```diff
@@ -262,9 +262,26 @@ class ValueId {
     return addDatatypeBits(bits, Datatype::Bool);
   }
 
+  /// Create a `ValueId` for a boolean value, represented as 0/1 instead of
+  /// false/true.
```

Comment:
```suggestion
  /// Create a `ValueId` for a boolean value, represented as "0" or "1" instead of
  /// "false" or "true".
```
And the name of the function is wrong, it should be `makeFromNumericBool` or even better `makeBoolFromZeroOrOne` (very strange semantics and only used in one place, so the name can be explicit.